client/src/containers/MoviePageContainer.jsx

```diff
@@ -1,14 +1,18 @@
-import {connect} from 'react-redux';
-import { withRouter } from 'react-router-dom';
+import { connect } from "react-redux";
+import { withRouter } from "next/router";
 
-import MoviePage from '../components/MoviePage';
+import MoviePage from "../components/MoviePage";
 
-const mapStateToProps = (state) => ({
-    selectedMovie:state.movieDetails.selectedMovie,
-    moviesOfTheSameGenre:state.movieDetails.moviesOfTheSameGenre
+const mapStateToProps = state => ({
+  selectedMovie: state.movieDetails.selectedMovie,
+  moviesOfTheSameGenre: state.movieDetails.moviesOfTheSameGenre
 });
 
-const mapDispatchToProps = () => ({
-});
+const mapDispatchToProps = () => ({});
```

Comment:
_trivial_

Since `mapDispatchToProps` in this case doesn't provide any valuable input in the codebase, please consider removing it.